spec/hotel_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'spec_helper'
+
+describe 'Hotel class' do
+  before do
+    @hotel = HotelBooking::Hotel.new(hotel_name: 'Wyndham', number_of_rooms: 20)
+    @hotel.rooms = []
+    @hotel.bookings = []
+  end
+
+  describe 'Hotel instantiation' do
+    it 'creates an instance of hotel' do
+      expect(@hotel).must_be_kind_of HotelBooking::Hotel
+    end
+
+    it 'is set up for specific attributes and data types' do
+      expect(@hotel.hotel_name).must_be_kind_of String
+      expect(@hotel.number_of_rooms).must_be_kind_of Integer
+      expect(@hotel.rooms).must_be_kind_of Array
+      expect(@hotel.bookings).must_be_kind_of Array
```

Comment:
This test is not terribly important. What the class is composed of is less interesting than what it can do.